### PR TITLE
AMP-75104 - add missing migration logic admonition

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

https://amplitude.atlassian.net/browse/AMP-75104

image

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: includes/sdk-missing-migration.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+!!! warning "Migration logic not available"
+    The maintenance SDK stores device ID, user ID, etc., in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the latest SDK. Consequently, for example, device ID may be regenerated when using the latest SDK.
+
```
Comment (yuhao900914) [inline includes/sdk-missing-migration.md:3]: For swift, the un-send event won't be migrated either.  We should also add that.

Comment (Mercy811) [reply includes/sdk-missing-migration.md:3]: Do we migrate un-send events for Android?

Comment (Mercy811) [reply includes/sdk-missing-migration.md:3]: It seems we don't. Will have this in Android Migration doc as well.

**Review by bgiori — APPROVED** [review]

File: includes/sdk-missing-migration.md (hunk 1 above)
Comment (justin-fiedler) [reply includes/sdk-missing-migration.md:3]: The migration plugin in Kotlin will send any pending events from storage

File: includes/sdk-missing-migration.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+!!! warning "Migration logic not available"
+    The maintenance SDK stores device ID, user ID, unsend events etc., in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the latest SDK. Consequently, for example, device ID may be regenerated when using the latest SDK and unsend events will be lost.
```
Comment (justin-fiedler) [inline includes/sdk-missing-migration.md:2]:
```suggestion
    The maintenance SDK stores device ID, user ID, unsent events, etc. in SQLite. However, there is no migration logic available (at this time), so this data won't be automatically migrated when upgrading to the latest SDK. Consequently, for example, device ID may be regenerated when using the latest SDK and unsend events will be lost.
```

File: includes/ampli/ampli-supported-platforms.md [hunk 3]
```diff
@@ -4,14 +4,14 @@ The other Amplitude SDKs for _Flutter_, _Unity_, and _Unreal_ _do not_ have Ampl
...
-    | Java         | Yes           | :octicons-package-16: [`com.amplitude:java-sdk`](../java/)<br/>:material-file-document:[Ampli documentation](../java/ampli/)                                                                                                                                                                                                                                                                                        |
+    | Browser      | Yes           | :octicons-package-16: [`@amplitude/analytics-browser`](/data/sdks/typescript-browser/)<br/>:material-file-document: [Ampli documentation](/data/sdks/typescript-browser/ampli/)<br/><br/>:material-hammer-screwdriver:{.red}:octicons-package-16: [`amplitude-js`](/data/sdks/javascript/)<br/>:material-hammer-screwdriver:{.red}:material-file-document:[Ampli documentation](/data/sdks/javascript/ampli/)                  |
```
Comment (justin-fiedler) [inline includes/ampli/ampli-supported-platforms.md:7]: Thank you @Mercy811 !

**Review by justin-fiedler — APPROVED** [review]

File: includes/ampli/ampli-supported-platforms.md (hunk 3 above)
Comment (Mercy811) [reply includes/ampli/ampli-supported-platforms.md:7]: no problem!

**Review by yuhao900914 — APPROVED** [review]

File: includes/sdk-missing-migration.md (hunk 1 above)
Comment (Mercy811) [reply includes/sdk-missing-migration.md:3]: Thanks Justin, I found the PR https://amplitude.atlassian.net/browse/AMP-52578

**Pull request merged**